integration/batching-with-context/batching.ts

```diff
@@ -239,12 +239,14 @@ export const BatchMapQueryResponse = {
   },
 
   fromJSON(object: any): BatchMapQueryResponse {
-    const message = createBaseBatchMapQueryResponse();
-    message.entities = Object.entries(object.entities ?? {}).reduce<{ [key: string]: Entity }>((acc, [key, value]) => {
-      acc[key] = Entity.fromJSON(value);
-      return acc;
-    }, {});
-    return message;
+    return {
+      entities: isObject(object.entities)
+        ? Object.entries(object.entities).reduce<{ [key: string]: Entity }>((acc, [key, value]) => {
+            acc[key] = Entity.fromJSON(value);
+            return acc;
+          }, {})
+        : {},
```

Comment:
Any reason for the `object.entities ?? {}` => `isObject() ? X : Y` change other than taste? it's probably a bit easier to review but is more code that has to be send to web users.

Reply:
Not sure, but I think I was trying to keep the pattern of "if there's a value, read the value, otherwise use the default", to make it more easily readable.

Reply:
Yeah, sure. The _iterate over an empty object_ approach is not the most obvious.